orca-webhook/src/main/groovy/com/netflix/spinnaker/orca/webhook/tasks/CreateWebhookTask.groovy

```diff
@@ -32,13 +31,19 @@ import com.netflix.spinnaker.orca.webhook.service.WebhookService
 import groovy.util.logging.Slf4j
 import org.apache.http.HttpHeaders
 import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.ResponseEntity
 import org.springframework.stereotype.Component
 import org.springframework.web.client.HttpStatusCodeException
 
+import java.util.regex.Matcher
+import java.util.regex.Pattern
+
 @Slf4j
 @Component
 class CreateWebhookTask implements RetryableTask {
 
+  private static final Pattern URL_SCHEME = Pattern.compile("(.*)://(.*)")
```

Comment:
was trying to see if there are ways to make the `URL` class do the "hard" work here but can't find anything.